docker-compose.yml

```diff
@@ -24,7 +24,7 @@ services:
         read_only: False
       - /fidesops/src/fidesops.egg-info
     environment:
-      - FIDESOPS__DEV_MODE=${FIDESOPS__DEV_MODE}
+      - FIDESOPS__DEV_MODE=True
```

Comment:
Used to set `developer` (`bool`) when calling fideslog. Our reasoning why we hard-code this to `true` here is because if someone is using our default docker-compose file, that would be considered dev/test mode.

Someone actually using this in prod would need to create their own `docker-compose.yml` with specific config per their environment, as detailed here- https://ethyca.github.io/fidesops/getting_started/